Dockerfile

```diff
@@ -68,12 +68,18 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
 COPY requirements.txt requirements.txt
 RUN pip install -r requirements.txt
 
+
+
 # Use renv for R packages
 WORKDIR /usr/local/renv
 ENV RENV_CONFIG_CACHE_ENABLED=FALSE
+RUN Rscript -e "install.packages('renv')"
+
+# Temporary fix for broken(?) RSamtools package
+RUN Rscript -e "install.packages('BiocManager'); BiocManager::install('Rsamtools')"
```

Comment:
ooof what happened to RSamtools?

Reply:
I'm not sure... something in an interaction between the Bioc docker image, `renv` and `Rsamtools` means that the package it downloads is corrupted and fails to untar properly. Ally ran into it in her Docker image a week or so ago. I'm expecting it to be resolved at some point when things rebuild somewhere.